posts/archive/taking-automation-to-the-next-level-using-ansible-gitops-to-manage-the-lifecycle-of-a-containerized-application.md

```diff
@@ -0,0 +1,497 @@
+---
+author: Roger Lopez
+date: 2022-07-06 00:00 UTC
+description: One of the great advantages of combining GitOps with
+  Ansible is that you get to streamline the automation delivery and the
+  lifecycle of a containerized application.
+lang: en-us
+title: Using Ansible and GitOps to Manage the Lifecycle of a Containerized Application
+---
+
+# Using Ansible and GitOps to Manage the Lifecycle of a Containerized Application
+
+[![taking automation to the next level
+blog](https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=1200&name=taking%20automation%20to%20the%20next%20level%20blog.png){width="1200"
+loading="lazy" style="width: 1200px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=600&name=taking%20automation%20to%20the%20next%20level%20blog.png 600w, https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=1200&name=taking%20automation%20to%20the%20next%20level%20blog.png 1200w, https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=1800&name=taking%20automation%20to%20the%20next%20level%20blog.png 1800w, https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=2400&name=taking%20automation%20to%20the%20next%20level%20blog.png 2400w, https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=3000&name=taking%20automation%20to%20the%20next%20level%20blog.png 3000w, https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=3600&name=taking%20automation%20to%20the%20next%20level%20blog.png 3600w"
```

Comment:
hubspot specific links, display broken. 
we have to search for `/hs-fs/` and `hubfs/`, as those link to hubspot assets.